content/docs/examples/videos.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Videos"
+description = "Tutorials and overviews published in video format. Be aware that a video
```

Comment:
Suggestion, please move the description content to line 8. Please replace the description with something like:

Tutorial and overview videos